yarpcconfig/kit.go

```diff
@@ -33,6 +34,8 @@ import (
 // Kit is an opaque object that carries context for the Configurator. Build
 // functions that receive this object MUST NOT modify it.
 type Kit struct {
+	HTTPFallbackHandler http.Handler
```

Comment:
Are we okay with people being able to modify this on the Kit? Should this
field be private with a public accessor?

Reply:
Is good idea.